agent/fw_drupal8.c

```diff
@@ -508,7 +578,14 @@ NR_PHP_WRAPPER(nr_drupal8_module_handler) {
    * for oapi so the hook vars this is referencing do not exist.
    *
    */
```

Comment:
Shouldn't this comment be removed? It looks like the temporary exclusion has been addressed.